lib/handlers/base.rb

```diff
@@ -29,7 +29,7 @@ def setting(name, options = {})
         settings[name] = options
 
         define_method(name) do
-          parser.parse(@settings[name])
+          parser.parse(@settings[name.to_s])
```

Comment:
I need to have a closer look why retrieving a model from SQL I get a `HashWithIndiferentAccess`  but not using fixtures where I retrieve a `Hash`.

Reply:
```
 Handler.first.attributes_before_type_cast["settings"]
  Handler Load (1.1ms)  SELECT "handlers".* FROM "handlers" ORDER BY "handlers"."id" ASC LIMIT $1  [["LIMIT", 1]]
=> "--- !ruby/hash:ActiveSupport::HashWithIndifferentAccess\nslack_token: xoxp-111111111111111\nchannel: \"#aaaaaaa\"\nmessage: The user {{ first_name }} {{ last_name }}.\n"                                                                               
```

I'm a little bit puzzled here....